subjects/devops/better-cat/README.md

```diff
@@ -0,0 +1,64 @@
+## better-cat
+
+### Instructions
+
+Create a script `better-cat.sh` that will behave similarly to the cat command, but with additional functionality.
+
+The script should accept any number of filenames as arguments, and it should print the content of each file to the standard output. If no arguments are provided, the script should print the content of all files in the current directory.
+
+The script should also support the following options:
+
+- -c: If this option is provided, the script should exclude all lines that begin with the # character.
+- -l: If this option is provided, the script should print the line number before each line of output.
+- -r: If this option is provided, the script should print a summary line at the end of the output, indicating the total number of lines and characters printed.
+
+If both -c and -l are provided, the line number should include only non-comment lines.
```

Comment:
I would highlight the `-c` and `-l` with ` or bold/italic

Reply:
Done.